src/version/version.go

```diff
@@ -0,0 +1,24 @@
+package version
```

Comment:
what do you think rename this module to "semantic" ? @esequielvirtuoso

Reply:
Agreed!